### PR TITLE
OADP-4453: GO-2024-2842: bump github.com/containers/image/v5 v5.30.2

### PR DESCRIPTION
**Pull request by kaovilai**
Signed-off-by: Tiger Kaovilai <tkaovila@redhat.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -157,49 +152,62 @@ require (
...
 	github.com/titanous/rocacheck v0.0.0-20171023193734-afe73141d399 // indirect
-	github.com/vbauerster/mpb/v8 v8.4.0 // indirect
-	go.mongodb.org/mongo-driver v1.11.3 // indirect
-	golang.org/x/mod v0.11.0 // indirect
-	golang.org/x/tools v0.8.0 // indirect
+	github.com/vbauerster/mpb/v8 v8.7.2 // indirect
+	go.mongodb.org/mongo-driver v1.14.0 // indirect
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.45.0 // indirect
+	go.opentelemetry.io/otel v1.19.0 // indirect
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.19.0 // indirect
```
Comment (kaovilai) [inline go.mod:201]: to fix
```
❯ go mod tidy 
go: finding module for package go.opentelemetry.io/otel/exporters/otlp/internal
go: github.com/konveyor/openshift-velero-plugin/velero-plugins/imagecopy imports
        github.com/containers/image/v5/transports/alltransports imports
        github.com/containers/image/v5/docker/daemon imports
        github.com/docker/docker/client tested by
        github.com/docker/docker/client.test imports
        github.com/docker/docker/testutil imports
        go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp imports
        go.opentelemetry.io/otel/exporters/otlp/otlptrace imports
        go.opentelemetry.io/otel/exporters/otlp/internal: module go.opentelemetry.io/otel/exporters/otlp@latest found (v0.20.1), but does not contain package go.opentelemetry.io/otel/exporters/otlp/internal
```

Comment (kaovilai) [reply go.mod:201]: using version from go.mod here
https://github.com/containers/image/blob/v5.30.2/go.mod#L133C2-L133C73

**Review by weshayutin — APPROVED** [review]
nice, cleaner than what I was doing. Thank you @kaovilai !
/LGTM

**Review by sseago — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**